nfs_afs_sync.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# rsync the files in the deployed directory to afs to be picked up by acr
+LOG="$(readlink -f "$(dirname -- "${BASH_SOURCE[0]}")")/sync_log.txt"
+SOURCE="/cds/group/pcds/epics-dev/screens/pydm/pmps-ui"
+DEST="/afs/slac/g/lcls/tools/pydm/display/xray/pmps-ui"
+
+echo "Sync at $(date)" >> "${LOG}"
+echo "from ${SOURCE}" >> "${LOG}"
+echo "to ${DEST}" >> "${LOG}"
+# Avoid git repo + random junk like screenshots
+rsync -rv ${SOURCE}/*.py ${SOURCE}/*.ui ${SOURCE}/*.yml ${SOURCE}/*.sh ${SOURCE}/*.sh ${SOURCE}/templates "${DEST}" >> "${LOG}"
```

Comment:
* FYI `git ls-files` is pretty neat for this purpose - but this works and there's no need to change it
`*.sh` appears to be listed twice (which is fine for rsync, so again no big deal)

Reply:
I couldn't quickly figure out how to pipe the ls-files output into rsync, I'm sure there's just a gap in my bash knowledge

double sh is definitely a typo